docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -1003,6 +1003,19 @@ If the provided `expression` is NULL, returns `altValue`, otherwise, returns `ex
 Where the parameter type is a complex type, for example `ARRAY` or `STRUCT`, the contents of the
 complex type are not inspected.
 
+### `NULLIF`
+
+Since: -
+
+```sql
+NULLIF(parameter1, parameter2)
```

Comment:
Do these parameters support any expression like IFNULL? If so, is it better to use `NULLIF(expr1, expr2)` instead?

Reply:
Yes it does, will do it.

Reply:
Changed arguments name as suggested